### ISSUE TITLE
Refactor to avoid `test` reserved word

### ISSUE THREAD

**Issue by kljensen** [exercism/gleam-test-runner#43]
As you know (obviously 😜!) `test` is reserved in the latest Gleam release. This should be changed here, e.g.

```
warning: Reserved word used
   ┌─ /Users/foo/src/github.com/exercism/gleam/exercises/concept/log-levels/build/packages/exercism_test_runner/src/exercism/test_runner.gleam:48:13
   │
48 │ fn run_test(test: Test) -> TestResult {
   │             ^^^^

The word `test` has been reserved for future use.
```

I encountered this running the exercism exercises locally with Gleam v0.33.0. I'm happy to take a whack at this.

**Comment by lpil** [comment]
Fantastic, thank you!

**Comment by kljensen** [comment]
Blocked by #45 for which there is a PR in #46

**Comment by kljensen** [comment]
> Blocked by #45 for which there is a PR in #46

Not sure what I was thinking here. Wasn't necessarily blocking on v0.32. Likely I had a brain fart.

**Comment by kljensen** [comment]
> Currently waiting on [JohnBjrk/gap#8](https://github.com/JohnBjrk/gap/issues/8)

Actually I think this is closed by #44.

**Comment by lpil** [comment]
Thank you!
